src/components/views/elements/Dropdown.js

```diff
@@ -88,8 +93,6 @@ export default class Dropdown extends React.Component {
         this.state = {
             // True if the menu is dropped-down
             expanded: false,
-            // The key of the selected option
-            selectedOption: this.props.initialSelectedOption || props.children[0].key,
             // The key of the highlighted option
             // (the option that would become selected if you pressed enter)
             highlightedOption: props.children[0].key,
```

Comment:
this will blow up if children isn't an array, or is empty - eg on empty search results?  we should probably do something better.